adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -57,7 +57,7 @@ - (id)init {
     _initializingDelegates = [NSMapTable mapTableWithKeyOptions:NSPointerFunctionsStrongMemory
                                                    valueOptions:NSPointerFunctionsWeakMemory];
     _bannerDelegates = [NSMapTable mapTableWithKeyOptions:NSPointerFunctionsStrongMemory
-                                             valueOptions:NSPointerFunctionsWeakMemory];
+                                             valueOptions:NSPointerFunctionsStrongMemory];
```

Comment:
In the testing of multiple Banners with same placementID, `NSPointerFunctionsWeakMemory` always leads the `_bannerDelegates ` becoming empty, because the value object of  `_bannerDelegates ` is released early.